backend/build.gradle

```diff
@@ -201,7 +201,7 @@ task testDbStart(type: Exec) {
 }
 
 task testDbStop(type: Exec) {
-    commandLine "docker", "compose", "--env-file", "../.env.test", "-p", "simple-report-tests", "-f", "../docker-compose.yml", "down"
+    commandLine "docker", "compose", "--env-file", "../.env.test", "-p", "simple-report-tests", "-f", "../docker-compose.yml", "down", "-v"
```

Comment:
whats the need for this? don't test `truncate` the db before running the tests anyway? is this to get around the checksum issues?